config/locales/en.yml

```diff
@@ -919,6 +919,8 @@ en:
       video: "Watch a 30 second video on posting research notes &raquo;"
     wikiRich:
       new_wiki: "New Wiki Page"
+      questionRich:
+      consider_sharing: "Consider sharing:"
   home_controller:
```

Comment:
Thanks @RReiso, this looks great so far!

The level of indentation here matters because this is a [YAML file](https://en.wikipedia.org/wiki/YAML). `consider_sharing` needs to be nested inside of `questionRich`.

So instead of:
```
 wikiRich:
      new_wiki: "New Wiki Page"
      questionRich:
      consider_sharing: "Consider sharing:"
```

We need:
```
    wikiRich:
      new_wiki: "New Wiki Page"
    questionRich:
      consider_sharing: "Consider sharing:"
```

Basically `questionRich` should have the same level of indentation as `wikiRich`.

Reply:
Thanks @noi5e for pointing this out, I have made another commit. I really appreciate your help with this.